nixos/modules/services/misc/mbpfan.nix

```diff
@@ -24,10 +24,10 @@ in {
       description = lib.mdDoc "If true, sets the log level to verbose.";
     };
 
-    aggressive = mkOption {
```

Comment:
Can we please add a `mkRemovedOptionModule` here.

Reply:
Maybe I should just enable this by default, not to break compatibility?

Reply:
You mean not remove the option?

Reply:
Yes. I've made some changes, can you review again?